packages/docs-site/src/library/pages/components/popover.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Popover
+description: Popover is a self-contained modal or dialogue that appears in context next to a trigger element.
+header: true
+draft: false
+---
+
+import { Popover, Tab, TabSet } from '@royalnavy/react-component-library'
+import DataTable from '../../../components/presenters/data-table'
+import CodeHighlighter from '../../../components/presenters/code-highlighter'
+import SketchWidget from '../../../components/presenters/sketch-widget'
+
+# Overview
+
+A popover is a self-contained modal or dialogue that appears in context next to a trigger element. It contains more content that a tooltip, but is directly related to a trigger element in the same way as a tooltip.
```

Comment:
`that` 👉  `than`

Reply:
Ooops - copied pasted from the issue text.